src/contentScript/contentScript.scss

```
@@ -40,8 +40,18 @@
   border-color: transparent !important;
   background: linear-gradient(
     20deg,
-    rgba(182, 109, 255, 0.4),
-    rgba(182, 109, 255, 0.1)
+    rgba(164, 73, 255, 0.4),
+    rgba(164, 73, 255, 0.1)
+  ) !important;
+}
```

Let's use nesting here instead of copy-pasting the long selector, comment and unrelated property:

```suggestion
&.light {
   background: linear-gradient(
     20deg,
     rgba(213, 170, 255, 0.4),
     rgba(213, 170, 255, 0.1)
   ) !important;
}
```